jack_sneed/test/test.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+var expect = require('chai').expect;
+var greet = require(__dirname + '/../lib/greet');
+
+describe('the greet function', function() {
+  it('the function should say hello (name)', function() {
+    expect(greet('jack')).to.eql('hello jack');
+  });
+});
```

Comment:
All good. One picky thing for future reference is to add one blank line at the end of the code. This is according to the code style  rules we use in this class.